tldraw/apps/tldraw-logseq/src/components/ActionBar/ActionBar.tsx

```diff
@@ -19,10 +22,12 @@ export const ActionBar = observer(function ToolBar(): JSX.Element {
 
   return (
     <div className="action-bar">
+      {/* <button>minimap</button> */}
+      <Minimap></Minimap>
       <button onClick={undo}>
         <UndoIcon></UndoIcon>
       </button>
-
+        <ZoomMenu></ZoomMenu>
```

Comment:
It seems your editor does not configured tab width correctly, which should be 2 spaces in files under `/tldraw`. Also, you may want to enable auto formatting with Prettier, whose config can be found in `tldraw/.prettierrc`